24.md

```diff
@@ -39,3 +39,4 @@ tags
 These tags may be present in multiple event kinds. Whenever a different meaning is not specified by some more specific NIP, they have the following meanings:
 
   - `r`: a web URL the event is referring to in some way
+  - `title`: title of the event
```

Comment:
By "event" here, you mean the calendar event, right?

Reply:
Correct.

Reply:
Well, not necessarily, it could also just be the title of a non-calendar Nostr event.

Reply:
Ok, thanks!